README.md

```diff
@@ -0,0 +1,34 @@
+# Dart
+
+[Dart][website] is an open-source, scalable programming language, with robust
+libraries and runtimes, for building web, server, and mobile apps.
+
+## Using Dart
+
+Visit the [Dart website][website] to learn more about the
+[language][lang], [tools][tools],
+[tutorials][codelab], and more.
+
+Browse [pub.dartlang.org][pubsite] for more packages and libraries contributed
+by the community and the Dart team.
+
+## Contributing to Dart
+
+The easiest way to contribute to Dart is to [file issues][dartbug].
+
+If you want to contibute a patch,
```

Comment:
contibute, ha! That's the typo I fixed in that other README. But how about we say:

You can also contribute patches, as described in [Contributing][contrib].